candi/openapi/doc-ru-node_group.yaml

```
@@ -10,14 +10,14 @@ spec:
               properties:
                 nodeType:
                   description: |
-                    Тип узлов, которые представляет эта группа:
+                    Типы узлов, которые представляет эта группа:
```

```suggestion
                    Типы узлов, которые представляет эта группа:
```
I have a question about this....